data/sts/semeval-sts/README.md

```diff
@@ -50,6 +50,8 @@ set.
 |                          |±0.032854 |±0.005099 |±0.007836 |±0.005869 |±0.010037 |±0.007489 |±0.005823 |±0.094360 |
 | attn1511                 | 0.712086 | 0.656483 | 0.429167 | 0.632170 | 0.628803 | 0.657264 | 0.668384 | 0.603158 | (defaults)
 |                          |±0.033190 |±0.009479 |±0.019904 |±0.016477 |±0.015415 |±0.012070 |±0.023045 |±0.109596 |
+| skipthoughts             | 0.713562 | 0.430110 | 0.359320 | 0.633636 | 0.617385 | 0.561543 | 0.796134 | 0.593603 | ``l2reg=0.001`` ``dropout=0.2`` ``use_eos=1`` ``use_flags=0``
+|                          |±0.000664 |±0.001639 |±0.001614 |±0.000804 |±0.001273 |±0.000864 |±0.000488 | ±0.031334
```

Comment:
Interesting - the performance patterns look quite different from our other models.  Any intuition why?  (Low prio question.)